ietf-layer0-types.yang

```diff
@@ -1339,15 +1504,15 @@ module ietf-layer0-types {
       description "Maximum Differential group delay of this mode
                   for this lane";
     }
-    list max-pol-dependent-loss-penalty {
+    list max-pdl-penalty {
       config false;
       description
         "Optional penalty associated with the maximum acceptable 
         accumulated polarization dependent loss.
          This list of pair pdl and penalty can be used to
          sample the function pdl = f(penalty).";
-      leaf max-polarization-dependent-loss {
-        type power-in-db-or-null;
+      leaf max-pdl-value {
```

Comment:
I suggest to remove the max from the penalty name and attributes (otherwise confusing)
list pdl-penalty {
...
leaf pdl-value

Reply:
Addressed in https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/70/commits/0038f5e771db23c9e2109f149b30d3226340db67

Co-authored-by: sergio belotti <sergio.belotti@nokia.com>